gazebo_ros2_control/include/gazebo_ros2_control/gazebo_ros2_control_plugin.hpp

```diff
@@ -48,6 +48,8 @@
 #include "gazebo/common/common.hh"
 #include "gazebo/physics/physics.hh"
 
+#include "gazebo_ros/node.hpp"
```

Comment:
This could be just included in the `cpp` file. It's not used by the header, so it shouldn't need to be exposed.

Reply:
moved https://github.com/ros-simulation/gazebo_ros2_control/pull/11/commits/4912b176b41705bc70a6fd8a2ccfaf987d87de02